packages/nuxt/src/app/components/nuxt-link.ts

```diff
@@ -299,10 +316,26 @@ export function defineNuxtLink (options: NuxtLinkOptions) {
       const el = import.meta.server ? undefined : ref<HTMLElement | null>(null)
       const elRef = import.meta.server ? undefined : (ref: any) => { el!.value = props.custom ? ref?.$el?.nextElementSibling : ref?.$el }
 
+      function checkShouldPrefetch () {
+        const shouldPrefetch = props.prefetch !== false && props.noPrefetch !== true && props.target !== '_blank' && !isSlowConnection() && !prefetched.value
+        return shouldPrefetch
+      }
+
+      async function prefetch (nuxtApp = useNuxtApp()) {
```

Comment:
```suggestion
      async function prefetch (nuxtApp = useNuxtApp()): Promise<void> {
```

Reply:
@quentinmcq I don't see any value with declaring the return type, inferring return type is fine